README.md

```diff
@@ -24,7 +24,7 @@ This is a **Node.js** SDK to make it easy to talk to the
 
 ## Requirements
 
-- [Node.js](https://nodejs.org/en/) version 18 or newer
+- [Node.js](https://nodejs.org/en/) version 20 or newer
```

Comment:
We were debating this in Slack, do we need to drop 18?

Reply:
we don't really need to drop it, but the advantage is that we will be able to update packages like `got` in the future without having to make another major release of node-sdk. and considering 20 is EOL this month, then that'll just be a few weeks of us not supporting an LTS version of node